NEWS.md

```diff
@@ -88,6 +88,13 @@
   ([#3081](https://github.com/JuliaData/DataFrames.jl/pull/3081))
 * Make `subset` preserves group ordering when `ungroup=false` like `subset!` already does
   ([#3094](https://github.com/JuliaData/DataFrames.jl/pull/3094))
+* Fix incorrect behavior of `GroupDataFrame` indexing in corner cases
+  ([#3179](https://github.com/JuliaData/DataFrames.jl/pull/3179))
+* Fix incorrect behavior of `insertcols!` when no columns to add are passed
+  ([#3179](https://github.com/JuliaData/DataFrames.jl/pull/3179))
```

Comment:
"Incorrect behavior" makes it sound like corruption could happen, but this just fixes an error, right?

Reply:
This fixes the following issues:
```
julia> df = DataFrame(a=1)
1×1 DataFrame
 Row │ a
     │ Int64
─────┼───────
   1 │     1

julia> insertcols!(df, :a)
ERROR: MethodError: insertcols!(::DataFrame, ::Symbol) is ambiguous.

julia> insertcols!(df, :a, after=true)
ERROR: MethodError: (::DataFrames.var"#insertcols!##kw")(::NamedTuple{(:after,), Tuple{Bool}}, ::typeof(insertcols!), ::DataFrame, ::Symbol) is ambiguous.

julia> insertcols!(df, 1, after=true)
ERROR: ArgumentError: inserting colums using a keyword argument is not supported, pass a Pair as a positional argument instead

julia> insertcols!(df, after=true)
ERROR: ArgumentError: inserting colums using a keyword argument is not supported, pass a Pair as a positional argument instead
```
After the PR we will have:
```
julia> df = DataFrame(a=1)
1×1 DataFrame
 Row │ a     
     │ Int64 
─────┼───────
   1 │     1

julia> insertcols!(df, :a)
1×1 DataFrame
 Row │ a     
     │ Int64 
─────┼───────
   1 │     1

julia> insertcols!(df, :a, after=true)
1×1 DataFrame
 Row │ a     
     │ Int64 
─────┼───────
   1 │     1

julia> insertcols!(df, 1, after=true)
1×1 DataFrame
 Row │ a     
     │ Int64 
─────┼───────
   1 │     1

julia> insertcols!(df, after=true)
1×1 DataFrame
 Row │ a     
     │ Int64 
─────┼───────
   1 │     1
```

The point is that currently we have:
```
julia> df = DataFrame(a=1)
1×1 DataFrame
 Row │ a
     │ Int64
─────┼───────
   1 │     1

julia> insertcols!(df, 1)
1×1 DataFrame
 Row │ a
     │ Int64
─────┼───────
   1 │     1

julia> insertcols!(df)
1×1 DataFrame
 Row │ a
     │ Int64
─────┼───────
   1 │     1
```
so other cases should work the same.

Given this - what do you propose to write in NEWS.md?

Reply:
I'd just say 
```suggestion
* Fix errors in `insertcols!` when no columns to add are passed
  ([#3179](https://github.com/JuliaData/DataFrames.jl/pull/3179))
```
to stress that there was no risk of invalid results in previous versions.

Maybe this applies to other points too?

Reply:
For `GroupedDataFrame` indexing incorrect results could happen. For `insertcols!` the same applies. I fixed it.